frontend/packages/ceph-storage-plugin/src/components/ocs-install/node-list.tsx

```diff
@@ -147,13 +157,44 @@ const CustomNodeTable: React.FC<CustomNodeTableProps> = ({ data, loaded, ocsProp
   const [error, setError] = React.useState('');
   const [inProgress, setProgress] = React.useState(false);
   const [selectedNodesCnt, setSelectedNodesCnt] = React.useState(0);
+  const [nodesWarningMsg, setNodesWarningMsg] = React.useState('');
 
   let storageClass = '';
 
+  const validateCPU = (node: formattedNodeType) => {
+    return Number(node.cpuCores) < MIN_NODE_CPU;
```

Comment:
You should not be doing any checking of the humanized values. Only check the raw value from the original node object and use the helpers in units.js to parse.

This assumes that the value is specified in cores. It could be using a different unit like millicores. If it is, the check here is wrong.